### PR TITLE
[www] Change starter showcase URL from /starter-showcase/ to /starters/

### LINKED ISSUE #8087: starter showcase - change the index page URL from /starter-showcase/ to /starters/

### PR DESCRIPTION
**Pull request by fk**
Fixes #8087.

### CODE REVIEW

**Comment by fk** [comment]
Ah yeah @DSchau, of course — good call on adding a redirect!

> Do we need to redirect these? I think it's probably fine, I don't think a lot of people were linking to specific sites!

We only need to redirect the one changed URL to the index page though, detail view URLs haven't changed.

---

While we're at that — I didn't think about redirects, but I did search the project for links to `/starter-showcase/` and found that we still link to https://dreamy-shannon-191f15.netlify.com/starter-showcase/?sort=stars and https://next.gatsbyjs.org/showcase/ from this blog post: https://www.gatsbyjs.org/blog/2018-07-07-graphic-design-class/#anything-they-didnt-like (look for "NOTE: His students are in luck! The Starter Showcase…" in that section to find the links).

IMO we shouldn't do either?!

File: www/gatsby-node.js [hunk 1]
```diff
@@ -48,6 +48,12 @@ exports.createPages = ({ graphql, actions }) => {
     toPath: `/docs/sourcing-from-netlify-cms`,
   })
 
+  createRedirect({
```
Comment (fk) [inline www/gatsby-node.js]: @pieh This should be fine? 🙏

Comment (pieh) [reply www/gatsby-node.js]: yeah, it should be

Comment (fk) [reply www/gatsby-node.js]: Thank you!

**Pull request merged**